dart-github-google/pubspec.yaml

```diff
@@ -1,14 +1,15 @@
 name: sample
-version: 0.0.1
+version: 0.1.0-dev
 description: A sample library.
 author: Foo Bar <foobar@example.com>
 homepage: https://github.com/google/sample
 
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=1.24.0 <2.0.0'
+
+dependencies:
+  # lib_name: "^1.2.0"
 
-#dependencies:
-#  lib_name: any
 dev_dependencies:
-# change this to a specific version range when you create your project
-  test: any
+  # Narrow this constraint if you use more advanced test features.
+  test: "^0.12.0"
```

Comment:
drop quotes?

Reply:
I'm  bit ambivalent. We should at least standardize on single-vs-double, though